pom.xml

```diff
@@ -416,7 +416,7 @@
             <dependency>
                 <groupId>io.prestosql.hive</groupId>
                 <artifactId>hive-apache</artifactId>
-                <version>2.3.4-1</version>
+                <version>3.0.0-1-SNAPSHOT</version>
```

Comment:
Hive 3.0 and 3.1 are quite different with regards to timestamp.
Are you going to upgrade to Hive 3.1 later?

Reply:
See: https://github.com/prestosql/presto/pull/198

Reply:
Yes, but this is an easier intermediate step, needed now to fix the linked Parquet issue. I’m still working through the 3.1 update.